### PR TITLE
API Level 14 Compatibility

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: android/widgets/src/main/java/org/nativescript/widgets/HorizontalScrollView.java [hunk 1]
```diff
@@ -191,11 +193,16 @@ private void scrollToChild(View child) {
...
     }
-    
+
+    @TargetApi(17)
     private boolean isLayoutRtl() {
-        return (this.getLayoutDirection() == LAYOUT_DIRECTION_RTL);
+        if (version >= 17) {
+            return (this.getLayoutDirection() == LAYOUT_DIRECTION_RTL);
+        } else {
+            return true;
```
Comment (hshristov) [inline android/widgets/src/main/java/org/nativescript/widgets/HorizontalScrollView.java:202]: This should be `false`. We don't want layout to be RTL on API<17

**Review by hshristov — CHANGES_REQUESTED** [review]
Is RTL should return false for API<17.

File: android/widgets/src/main/java/org/nativescript/widgets/HorizontalScrollView.java (hunk 1 above)
Comment (ghost) [reply android/widgets/src/main/java/org/nativescript/widgets/HorizontalScrollView.java:202]: @hshristov I no longer have this repo and I can't edit it here

**Comment by vakrilov** [comment]
Closing due to innactivity

**Pull request closed without merging**